client/components/Registration/Register.jsx

```diff
@@ -1,94 +1,112 @@
-import React, { useState } from 'react'
+import React from 'react'
+// import { useFormik } from 'formik'
+import { useFormik } from 'formik'
 import { useHistory } from 'react-router-dom'
 import { registerUser } from './registerHelper'
 import { useAuth0 } from '@auth0/auth0-react'
+import * as Yup from 'yup'
 
-export function Register () {
+const registerSchema = Yup.object().shape({
+  firstName: Yup.string()
+    .min(2, 'Too Short!')
+    .max(15, 'Too Long!')
+    .required('Required'),
+  lastName: Yup.string()
+    .required('Required')
+    .min(2, 'Too Short!')
+    .max(15, 'Too Long!'),
+  username: Yup.string()
+    .min(2, 'Too Short!')
+    .max(15, 'Too Long!')
+    .required('Required'),
+  gardenId: Yup.number()
+    .required('Required')
+})
```

Comment:
Update these error messages to help the user be more successful. For example, instead of "Too Short!", you might use "This value must be at least 2 characters long". Don't forget to update the tests that check for these messages.